Cargo.toml

```diff
@@ -21,6 +21,7 @@ canonical_derive = {version = "0.5", optional = true}
 [dev-dependencies]
 rand = "0.7"
 anyhow = "1.0"
+lazy_static = "1.0"
```

Comment:
Shouldn't be "1.4"?

Reply:
Since this is already using major versions, 1.4 will automatically be taken by cargo.
See:
```
schnorr on  update_tests is 📦 v0.6.0 via 🦀 v1.52.0-nightly 
✦ ➜ cargo tree | grep lazy
│   │   │   │   │   └── lazy_static v1.4.0
│   │   │   │   ├── lazy_static v1.4.0 (*)
│   │       ├── lazy_static v1.4.0 (*)
│   │   │   └── lazy_static v1.4.0 (*)
│   │   └── lazy_static v1.4.0 (*)
├── lazy_static v1.4.0 (*)
```